src/locales/en/translation.json

```diff
@@ -341,9 +335,8 @@
             "UK biometric residence permit (BRP)"
           ]
         },
-        "subHeading3": "What would you like to do?",
+        "subHeading3": "How would you like to prove your identity?",
         "formRadioButtons": {
-          "radioButtonBackToRpText": "Go to the service you need to use and find out if they take other types of ID",
           "radioButtonBackToRpP1Text": "Go to the service you need to use and look for other ways to prove your identity",
```

Comment:
I guess make this `radioButtonBackToRpText` now?

Reply:
Done thanks